gtfu/src/main/java/gtfu/tools/Slack.java

```diff
@@ -0,0 +1,33 @@
+package gtfu.tools;
+import gtfu.HTTPClient;
+import gtfu.Debug;
+
+/**
+ * Interact with Slack
+ * @see <a href="https://slack.dev/java-slack-sdk/guides/incoming-webhooks</a>
+ */
+public class Slack {
+    private String webHookURL;
+
+    /**
+     * Initialize Slack connection
+     */
+    public Slack() {
+        this.webHookURL = System.getenv("GRAAS_BOT_WEBHOOK");
+        if(webHookURL == null){
```

Comment:
If `webhookURL` is null, then it's over. Nothing will work, so we should do more than just do a `Debug.log()` Remember that debug logging may be turned off in production. We should at least do `Debug.error()` and throw an exception.

Reply:
thanks, done!